src/coreComponents/fileIO/silo/SiloFile.cpp

```diff
@@ -1413,7 +1413,7 @@ void SiloFile::WriteElementMesh( ElementRegionBase const & elementRegion,
 
     int count = 0;
 
-    elementRegion.forElementSubRegions( [&]( auto const & elementSubRegion )
+    elementRegion.forElementSubRegions< CellElementSubRegion, FaceElementSubRegion, WellElementSubRegion >( [&]( auto const & elementSubRegion )
```

Comment:
```suggestion
    elementRegion.forElementSubRegions< CellElementSubRegion, 
                                        FaceElementSubRegion, 
                                        WellElementSubRegion >( [&]( auto const & elementSubRegion )
```
I assume this is to exclude the embedded elements?

Reply:
Yes, there is only vtk output for those.